jsapp/scss/components/_kobo.multi-checkbox.scss

```diff
@@ -0,0 +1,18 @@
+.multi-checkbox {
+  list-style: none;
+  height: auto;
+  position: absolute;
+  top: 20px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  overflow: hidden auto;
+  border: 1px solid $kobo-lightgray;
+  border-radius: 2px;
+  margin-top: 6px;
```

Comment:
I think you need to drop `position`, `top`, `left`, `right`, `bottom` and `margin-top` as these should be set by parent component depending on context (e.g. for Project Downloads we have flexbox layouting, not absolute positioning)